### PR TITLE
Running online events with Open Source communication tools

### PR DESCRIPTION
**Pull request by benparsons**

### CODE REVIEW

File: gatsby/content/docs/online-events.mdx [hunk 1]
```diff
@@ -19,8 +19,8 @@ recreate the fun and atmosphere of a real meetup in a decentralised way.
 ## Use Jitsi for Video
 
 It's important to us, wherever possible, to use Open tools. Jitsi Meet is Open 
-Source, self-hostable video conferencing platform. [Installing Jitsi is a 
-much-documented experience](https://github.com/jitsi/jitsi-meet/blob/master/doc/
+Source, self-hostable video conferencing platform.
+[Installing Jitsi is a much-documented experience](https://github.com/jitsi/jitsi-meet/blob/master/doc/
```
Comment (babolivier) [inline gatsby/content/docs/online-events.mdx:23]: This still has a new line in the middle of the URL (which is the issue, md doesn't really care about having a line break in the middle of the text).

File: gatsby/content/docs/online-events.mdx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+First, we want to get our speakers all connected. To do this, we created a new 
+room on Jitsi (the name doesn't matter), and made sure to add a password, since 
+this is not the default. All the speakers for the session will connect to this 
+room.
+
+![add a password to a jitsi room](/docs/img/jitsi-add-password.png)
+
+Next, we configured the Jitsi Meet server to stream to YouTube, and also made 
+the stream available directly via a video control on https://stream.matrix.org.
```
Comment (babolivier) [inline gatsby/content/docs/online-events.mdx:35]: I feel like this is going to make a lot of people ask about how we did that, so probably worth elaborating on that point?

Comment (benparsons) [reply gatsby/content/docs/online-events.mdx:35]: I agree. For now I'm waiting on time being available from the person or people who set it up.

File: gatsby/content/docs/online-events.mdx (hunk 1 above)
Comment (benparsons) [reply gatsby/content/docs/online-events.mdx:23]: Good spot

**Comment by benparsons** [comment]
Putting this live, we can edit later

**Pull request merged**